windows-calico/quickstart.md

```diff
@@ -103,11 +103,13 @@ The following steps install a Kubernetes cluster on a single Windows node, with
   <%
 
 1. Ensure that BGP is disabled.
+   If you installed using tigera-operator, you can do this by:
```

Comment:
`If you installed Calico using operator`?

Reply:
```suggestion
   If you installed Calico using operator, you can do this by:
```